app/controllers/users/invitations_controller.rb

```diff
@@ -212,13 +199,14 @@ def check_captcha_for_invite
 
     def check_invite_users_permission
       @user = current_user
-      @emails = params[:emails].map(&:downcase)
+      @emails = params[:emails]&.map(&:downcase)
       @team = Team.find_by_id(params['teamId'])
       @role = params['role']
 
-      render_403 if @emails && @emails.empty?
-      render_403 if @team && !can_manage_team_users?(@team)
-      render_403 if @role && !UserTeam.roles.keys.include?(@role)
+      render_403 unless @emails && @team && @role
+      render_403 if @emails.empty?
+      render_403 unless can_manage_team_users?(@team)
+      render_403 unless UserTeam.roles.key?(@role)
```

Comment:
what will happen if 2 or more conditions will match?

Reply:
Looks like it works, added `return` for safety reasons.